### PR TITLE
Rename getter for boolean fields from getXXX to isXXX

### LINKED ISSUE #2844: Rename getter for boolean fields from getXXX to isXXX
Consider changing the following convention for entities:
class X
{ 
   private Boolean a;
   ..
   public getA(){ return a};
}

to

class X
{ 
   private Boolean a;

   public isA(){ return a};
}

### PR DESCRIPTION
**Pull request by notflorian**
Fix #2844

### CODE REVIEW

File: generators/entity/templates/src/main/java/package/domain/_Entity.java [hunk 1]
```diff
@@ -160,7 +160,11 @@ public void setId(<% if (databaseType == 'sql') { %>Long<% } %><% if (databaseTy
     }
 <% for (fieldId in fields) { %>
     <%_ if (fields[fieldId].fieldTypeBlobContent != 'text') { _%>
+      <%_ if (fields[fieldId].fieldType.toLowerCase() == 'boolean') { _%>
```
Comment (deepu105) [inline generators/entity/templates/src/main/java/package/domain/_Entity.java]: indent

File: generators/entity/templates/src/main/java/package/domain/_Entity.java [hunk 2]
```diff
@@ -160,7 +160,11 @@ public void setId(<% if (databaseType == 'sql') { %>Long<% } %><% if (databaseTy
...
+    public <%= fields[fieldId].fieldType %> is<%= fields[fieldId].fieldInJavaBeanMethod %>() {
+      <%_ } else { _%>
```
Comment (deepu105) [inline generators/entity/templates/src/main/java/package/domain/_Entity.java]: indent 4 space

**Comment by notflorian** [comment]
I've updated _Entity.java to correct the indentation with 4 spaces.

I'm quite new with Github, is there a cleaner way to submit a push request with only the commits associated with my modifications, and not the commits of the merges with the master branch? Maybe with cherry picking?

**Comment by notflorian** [comment]
I succeded to clean unwanted commit with `git rebase upstream/master` :)

**Comment by deepu105** [comment]
you can rebase and squash commits if you want

**Pull request merged**